modules/storage-ephemeral-storage-strategy.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * storage/application-ephemeral-storage.adoc
+
+[id='strategy-{context}']
+= Strategy
+
+. Determine expected container ephemeral usage.
++
+If your administrator enabled the ephemeral storage technology preview, 
+determine expected mean and peak container ephemeral storage usage, 
+empirically if necessary, for example, by separate extended. Remember to 
+consider all the processes that might potentially run in parallel in the 
+container. For example, does the main application spawn any ancillary 
+scripts that may require local storage for work files or logging?
+
+. Assess risk for eviction.
++
+Determine the risk appetite for eviction. If the risk appetite is low, 
```

Comment:
I'd define risk appetite. I don't see a style entry for it, but it sounds like borderline jargon that might not translate well.